src/components/PlanPage/PlanPage.js

```diff
@@ -1,13 +1,40 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import ActivitiesListFilter from '../ActivitiesListFilter';
-import Week from '../Week/Week';
+import Week from '../Week';
+import { startSetActivities } from '../../store/activities/actions';
 
-const PlanPage = () => (
-  <div>
-    <ActivitiesListFilter data-test="filter-component" />
-    <Week data-test="week-component" />
-  </div>
-);
+class PlanPage extends React.Component {
+  static propTypes = {
+    onStartSetActivities: PropTypes.func,
+  };
 
-export default PlanPage;
+  static defaultProps = {
+    onStartSetActivities: () => {},
+  };
+
+  componentDidMount() {
+    this.props.onStartSetActivities();
```

Comment:
simply saying, this component seems to be like my `DashboardPage` from previous comment.